specification/network/resource-manager/Microsoft.Network/stable/2024-03-01/webapplicationfirewall.json

```diff
@@ -948,6 +955,75 @@
         }
       }
     },
+    "ExceptionEntry": {
+      "required": [
+        "matchVariable",
+        "valueMatchOperator"
+      ],
+      "description": "Adds exception to allow a request when the condition is satisfied.",
+      "properties": {
+        "matchVariable": {
+          "type": "string",
+          "enum": [
+            "RequestURI",
+            "RemoteAddr",
+            "RequestHeader"
+          ],
+          "description": "The variable on which we evaluate the exception condition",
+          "x-ms-enum": {
+            "name": "ExceptionEntryMatchVariable",
```

Comment:
WAF already has existing widely used feature that have similar properties - match variable, selector, operators namely, custom rules and exclusions. Exception is making use of those familiar properties. So, I don't think this would be hard to understand for the users.

If you would like to know more about the difference between Exception (previously known as Allowlist), custom rules and exclusion, we can have a discussion on it.